integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/longstring/VarcharDataTypesBasicTestCase.scala

```diff
@@ -191,6 +191,29 @@ class VarcharDataTypesBasicTestCase extends QueryTest with BeforeAndAfterEach wi
     assert(exceptionCaught.getMessage.contains("both in no_inverted_index and long_string_columns"))
   }
 
+  test("test alter table add long string columns with local dictionary") {
+    sql("drop table if exists testlongstring")
+    sql(
+      s"""
+         | CREATE TABLE if not exists testlongstring(id INT, name STRING, description STRING
+         | ) STORED BY 'carbondata'
+         |""".stripMargin)
+    sql("insert into testlongstring select 1, 'ab', 'cool'")
+    checkAnswer(sql("select * from testlongstring"), Seq(Row(1, "ab", "cool")))
+    checkExistence(sql("describe formatted testlongstring"), false, "long_string_columns")
+    //Alter table add table property of long string columns
+    sql("ALTER TABLE testlongstring SET TBLPROPERTIES('long_String_columns'='name,description')")
+    sql("insert into testlongstring select 1, 'ab1', 'not cool'")
```

Comment:
it's better to insert long string value, for example: `"123" * 12000`

Reply:
I have verified with long string, generating longstring also takes some time. so I didn't wanted to impact CI running time.

I will add for 33000 length, which is greater than short

Reply:
IMO, can use below code to insert data and check data:
```
    sql("""insert into testlongstring select 1, "abc" * 12000, 'not cool'""")
    checkAnswer(sql("select * from testlongstring"),
      Seq(Row(1, "ab", "cool"), Row(1, "abc" * 12000, "not cool")))
```
will it take a long time to run CI ?

By the way, it better to add a test case for unsetting long string column, if the length of value of long string column is more than 32000.

Reply:
ok, I will add for 33000 length string